server/queries/index.js

```diff
@@ -38,6 +39,19 @@ export default class Queries {
       })
   }
 
+  getPrrrsPerWeek(week){
```

Comment:
checkout the `metricsForWeek` method as well as the `Metrics` class to ensure its getting the data you need for the timeline. I think it already is. So we can remove this query too.